### PR TITLE
TVB-2936: Remove files and code related to portlets.

### PR DESCRIPTION
**Pull request by VinczeRobert**
If you search in the code for the word 'portlet' now you can see it only in RELEASE_NOTES and DeveloperReferenceManual.rst. I didn't do an extensive UI testing but I went through each page and the main functionalities and nothing seems to be broken.

### CODE REVIEW

File: framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/time_series/scripts/timeseriesSVG.js [hunk 1]
```diff
@@ -105,21 +93,15 @@ function _updateScalingFromSlider(value){
  * @param dt: time increment
  * @param channelLabels: a list with the labels for all the channels
  */
-function initTimeseriesViewer(baseURL, isPreview, dataShape, t0, dt, channelLabels, filterGid) {
+function initTimeseriesViewer(baseURL, dataShape, t0, dt, channelLabels, filterGid) {
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/time_series/scripts/timeseriesSVG.js:97]: `isPreview` param is still used in this function, breaking the TS D3 viewer

File: framework_tvb/tvb/interfaces/web/templates/jinja2/burst/main_burst.html [hunk 2]
```diff
@@ -40,7 +40,7 @@ <h4><span id="title-simulation"><mark>Configure</mark> New simulation core</span
 		<h4><mark>Parameter Space Exploration</mark> <span id="title-pse"></span></h4>
 		<a href="#" title="Maximize View" class="column-control action action-zoom-in" id="button-maximize-iso"
 			onclick="toggleMaximizeBurst(this);">Maximize</a>
-		<ul id="section-portlets-ul-pse" class="view-switcher tabswitcher">
+		<ul id="section-ul-pse" class="view-switcher tabswitcher">
 			<li id="pse-flot" class="active">
```
Comment (popaula937) [inline framework_tvb/tvb/interfaces/web/templates/jinja2/burst/main_burst.html:44]: I think section for PSE viewers can also be removed from here

**Review by popaula937 — COMMENTED** [review]
Please double-check at least the viewers that have changes in the JS side, as TS D3 viewer is broken

File: framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/pse_discrete/pse_discreet.js [hunk 3]
```diff
@@ -676,13 +676,6 @@ function d3Plot(placeHolder, data, options, pageParam) {
         .on("mouseover", function (d) {
             let offsetX = window.event.pageX;
             let offsetY = window.event.pageY - 100;
-            const portletContainer = document.getElementById("section-pse");
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/pse_discrete/pse_discreet.js:679]: I don't see any difference if I keep it there (I looked in the pse discrete visualizer, I thought that's where I should) but just for safety I will keep the code and name the constant pseContainer.

File: tvb_documentation/handouts/Node_01/Handout_UI_BuildingYourOwnBrainNetworkModel.tex [hunk 4]
```diff
@@ -312,7 +312,6 @@ \subsection{Building A Discrete Brain Network Model}\label{sec:region_simulation
 \begin{enumerate}[resume]
   \setcounter{enumi}{7}
  \item Provide the \underline{simulation length}, that here we'll use the default value of \textbf{\unit[1000]{ms}}.
- \item Before launching the simulation, configure a set of \underline{Visualizers} and/or \underline{Analyzers} by clicking on \underline{Configure}, selecting the what you want to see and saving your choices. (see Table \ref{tab:portletstab}). These windows will enable you to have a glimpse of the results as soon as the simulation ends.
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply tvb_documentation/handouts/Node_01/Handout_UI_BuildingYourOwnBrainNetworkModel.tex:315]: Okay

File: tvb_documentation/handouts/Node_02/Handout_UI_BrainNetworkModelsAtRest.tex [hunk 5]
```diff
@@ -318,7 +318,6 @@ \subsection{Building A Discrete Brain Network Model}\label{sec:region_simulation
 \begin{enumerate}[resume]
   \setcounter{enumi}{7}
  \item Provide the \underline{simulation length}, that here we'll use the default value of \textbf{\unit[1000]{ms}}.
- \item Before launching the simulation, configure a set of \underline{Visualizers} and/or \underline{Analyzers} by clicking on \underline{Configure}, selecting the what you want to see and saving your choices. (see Table \ref{tab:portletstab}). These windows will enable you to have a glimpse of the results as soon as the simulation ends.
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply tvb_documentation/handouts/Node_02/Handout_UI_BrainNetworkModelsAtRest.tex:321]: Okay

File: framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/time_series/scripts/timeseriesSVG.js [hunk 6]
```diff
@@ -26,17 +26,6 @@ var MAX_INITIAL_CHANNELS = 20;
 var _initial_magic_fcs_amp_scl;
 var tsView;
 
-/**
- * Initialize selected channels to default values. This is used when no selection ui is present (portlet mode)
- * initializes global TS_SVG_selectedChannels
- * Takes the first 20 channels or all
- */
-function _initDefaultSelection(){
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/time_series/scripts/timeseriesSVG.js:34]: I don't know, the comment above it said that "This is used when no selection ui is present (portlet mode)

[the remark this answers is not in the document]
Comment (VinczeRobert) [reply framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/time_series/scripts/timeseriesSVG.js:34]: I will readd it, it probably doesn't hurt anyways.

**Comment by popaula937** [comment]
Also, before merging this, it would be nice to squash the commits into a single one, so we could easily find the portlets cleanup in case it will be necessary.

File: framework_tvb/tvb/tests/framework/core/base_testcase.py [hunk 7]
```diff
@@ -97,8 +97,7 @@ class BaseTestCase(object):
     """
     This class should implement basic functionality which is common to all TVB tests.
     """
-    EXCLUDE_TABLES = ["ALGORITHMS", "ALGORITHM_CATEGORIES", "PORTLETS",
-                      "MAPPED_INTERNAL__CLASS", "MAPPED_MAPPED_TEST_CLASS"]
+    EXCLUDE_TABLES = ["ALGORITHMS", "ALGORITHM_CATEGORIES", "MAPPED_INTERNAL__CLASS", "MAPPED_MAPPED_TEST_CLASS"]
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply framework_tvb/tvb/tests/framework/core/base_testcase.py:100]: No

File: framework_tvb/tvb/core/entities/model/model_burst.py [hunk 8]
```diff
@@ -126,9 +74,6 @@ class BurstConfiguration(HasTraitsIndex):
     BURST_FINISHED = 'finished'
     BURST_CANCELED = 'canceled'
 
-    # TODO: Fix attrs for portlets
-    nr_of_tabs = 0
-    selected_tab = -1
```
[the remark this answers is not in the document]
Comment (VinczeRobert) [reply framework_tvb/tvb/core/entities/model/model_burst.py:131]: I don't think it is used for anything at this point but I could be wrong. I checked the TS viewers and the Simulation Page and everything seems fine without it.

**Comment by popaula937** [comment]
Having a burst selected on the history section displays results on the right-side section. After a browser Refresh, the results are not displayed anymore, as they were before these changes. Maybe because of some JS code being removed, please check and fix this.

**Pull request merged**